### PR TITLE
[CODE UNIFICATION] Use the PrebidRenderingConfig class instead of OriginalSDKConfigurationProtocol

### LINKED ISSUE #540: [CODE UNIFICATION] Confirm the PrebidRenderingConfig class to OriginalSDKConfigurationProtocol protocol
# Motivation

This task is related to the second step of targeting and bid requesting unification. (#431)

# Objectives

- Confirm the class `PrebidRenderingConfig` from the rendering API to the transitional protocol `OriginalSDKConfigurationProtocol`. 
- If `PrebidRenderingConfig` already has the required property or method but with a different name - Rename the item according to `OriginalSDKConfigurationProtocol` entry.
- Update unit tests

### PR DESCRIPTION
**Pull request by OlenaPostindustria**
Closes #540, #544

### CODE REVIEW

**Review by ValentinPostindustria — APPROVED** [review]
LGTM

File: Example/PrebidDemo/PrebidDemoObjectiveC/ViewController.m [hunk 1]
```diff
@@ -207,7 +207,7 @@ -(void) setStoredResponse {
 }
 
 -(void) setRequestTimeoutMillis {
-    Prebid.shared.timeoutMillis = 5000;
+    Prebid.shared.bidRequestTimeoutMillis = 5000;
```
Comment (YuriyVelichkoPI) [inline Example/PrebidDemo/PrebidDemoObjectiveC/ViewController.m:210]: Let's rename this property back.

File: InternalTestApp/PrebidMobileDemoRendering/AppDelegate.swift [hunk 2]
```diff
@@ -43,7 +43,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
 
         //Set up SDK.
-        PrebidRenderingConfig.initializeRenderingModule()
+        Prebid.initializeModule()
```
Comment (YuriyVelichkoPI) [inline InternalTestApp/PrebidMobileDemoRendering/AppDelegate.swift:46]: Let's rename to `initializeSDK()`

File: PrebidMobile/AdUnits/AdUnit.swift [hunk 3]
```diff
@@ -128,7 +129,7 @@ import ObjectiveC.runtime
             }
         }
 
-        DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(Prebid.shared.timeoutMillisDynamic), execute: {
+        DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(Int(truncating: Prebid.shared.bidRequestTimeoutDynamic ?? NSNumber(value: .PB_Request_Timeout))), execute: {
```
Comment (YuriyVelichkoPI) [inline PrebidMobile/AdUnits/AdUnit.swift:132]: let's rename this property back to `timeoutMillisDynamic`

File: PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/Prebid.swift [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+ limitations under the License.
+ */
+
+import Foundation
+import OMSDK_Prebidorg
+
+fileprivate let defaultTimeoutMillis = 2000
+
+@objcMembers
+public class Prebid: NSObject {
```
Comment (YuriyVelichkoPI) [inline PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/Prebid.swift:22]: let's move this file into some common place. For example, we can create a new folder in the root codebase directory - `ConfigurationAndTargeting` for this purpose. And move all respective files into this folder eventually.

**Review by YuriyVelichkoPI — CHANGES_REQUESTED** [review]

**Review by YuriyVelichkoPI — APPROVED** [review]
LGTM

**Pull request merged**